src/app/components/modal/index.html

```diff
@@ -179,6 +168,25 @@
 				<li>The form supports a complex task for a list of objects such as filling out report cards for a class.</li>
```

Comment:
Similar question here. The fourth bullet does not match the corresponding fourth bullet at https://developer.blackbaud.com/skyux/design/guidelines/form-design#form-containers. All the other 5 bullets match up. Should this bullet match as well? If so, how? @Blackbaud-ToddRoberts, should this bullet be copied to that corresponding list and the fourth bullet there copied here? Of should one of the fourth bullets replace the other? Seems like something is out of whack here, but I'm not sure how to fix it.

Reply:
I'm not sure which wording on those fourth bullets is better - they are trying to convey the same info. I assume one of them was wordsmithed, so whichever one you prefer can be the winner.

Reply:
@Blackbaud-DeniseCisneros, can you update this to consistently use whichever text you prefer? (Just my 2 cents, but I prefer the wording at https://developer.blackbaud.com/skyux/design/guidelines/form-design. But it's up to you which you think we should use.)

Reply:
I like the wording on the form design page too and updated the bullet with that text.

Reply:
Looks good. Thanks!